lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -158,6 +158,10 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
         system_identifier = INTERNAL_SYSTEM)
 
     persisted_resource_entries = [patient_entry, *location_resource_entries]
+    
+    REDCAP_RECORD_ID_CUTOFF = 32024
+    MIGRATED_RECORD = True if int(enrollment['record_id']) <= REDCAP_RECORD_ID_CUTOFF else False
```

Comment:
Might be safer to use `int(enrollment.get('record_id'))` here

Reply:
Either will throw an Exception if 'record_id' isn't present. `int(enrollment['record_id'])` will throw a KeyError:
```
Keyerror: 'record_id'
```
whereas `int(enrollment.get('record_id'))` will throw TypeError:
```
int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
I think the first one is more appropriate error in this case since there should always be a `record_id` on these.